pkg/cli/admin/release/extract_tools.go

```diff
@@ -370,12 +370,17 @@ func (o *ExtractOptions) extractCommand(command string) error {
 	releaseArch := release.Config.Architecture
 	releaseName := release.PreferredName()
 	refExact := release.ImageRef
-	refExact.Ref.Tag = ""
 	// if the release image is manifestlist image, we'll not change digest with
 	// arch based sha. Because we want that the extracted tool can be used for all archs.
 	if !release.IsManifestList {
 		refExact.Ref.ID = release.Digest.String()
+	} else {
+		// if the image is manifestlisted and referenced by tag, use the manifestlist digest.
+		if refExact.Ref.Tag != "" {
```

Comment:
I wonder if we can remove that conditional. What happened if we do this any case;

```
refExact.Ref.ID = release.ManifestListDigest
```

Reply:
it shouldn't matter as it's the same digest..i'll make the change